spec-main/chromium-spec.ts

```diff
@@ -375,7 +376,8 @@ ifdescribe(!process.env.IS_ASAN)('command line switches', () => {
     ifit(process.platform === 'linux')('should not change LC_ALL when --lang is not set', async () => testLocale('', lcAll, true));
   });
 
-  describe('--remote-debugging-pipe switch', () => {
+  // TODO(nornagon): figure out why these tests fail under ASan.
+  ifdescribe(!isAsan)('--remote-debugging-pipe switch', () => {
```

Comment:
I think it would be better manually removing all `ifdescribe(!process.env.IS_ASAN)` rather than reverting the commits, there are a few changes that should be kept.

For example tests like this can be enabled for asan, and the comment can be removed, and some tests were moved to `spec-main/fixtures/crash-cases` and it would be better keep them there.